common/baseDBMethods.js

```diff
@@ -0,0 +1,51 @@
+async function list(ctx, model) {
```

Comment:
Do you think that this file fits better in `routes/` directory?
You are right that it's common functionality, but it is more of a common functionality **between the routers**, since it handles routing functionality (as in `ctx.ok()`, `ctx.internalServerError()`, etc.).

Reply:
I don't mind, I will move it there.